pkg/endpoint/bpf.go

```diff
@@ -745,6 +745,9 @@ func (e *Endpoint) runPreCompilationSteps(regenContext *regenerationContext) (he
 		}
 		// Clean up map contents
 		e.getLogger().Debug("flushing old PolicyMap")
+		if dbgLog := e.getPolicyLogger(); dbgLog != nil {
```

Comment:
Instead of these scattered `nil` checks, can we have a function that takes `fields` and a string to log, and logs a debug message if the logger isn't `nil`?

Reply:
Done ;-)